docs/reclaimspace.md

```diff
@@ -101,6 +101,29 @@ data-pvc-1642664617   */1 * * * *                                   3s
 [go_cron]:
     https://pkg.go.dev/github.com/robfig/cron/v3#hdr-CRON_Expression_Format
 
+## Annotating StorageClass
+
+You can create `ReclaimSpaceCronJob` CR automatically by adding the
+`reclaimspace.csiaddons.openshift.io/schedule: "@midnight"` annotations to the
+StorageClass object. This will only inspect new PersistentVolumeClaims created from
+this StorageClass for ReclaimSpace operations. You must restart the controller if you
+want kubernetes-csi-addons to consider existing PersistentVolumeClaims.
+
+```console
+$ kubectl get storageclass rbd-sc
+NAME       PROVISIONER        RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
+rbd-sc     rbd.csi.ceph.com   Delete          Immediate           true                   5d2h
+
+$ kubectl annotate storageclass rbd-sc "reclaimspace.csiaddons.openshift.io/schedule=@midnight"
+storageclass.storage.k8s.io/rbd-sc annotated
+```
+
+**Note** Please note that the PersistentVolumeClaim annotation takes priority
+over StorageClass annotation. The kubernetes-csi-addons only generate a
```

Comment:
You need to document the priority of StorageClass and Namespace annotations too.